scripts/release/04-publish-release

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+if [[ "$#" -ne 1 ]]; then
+  echo "$0: VERSION"
+  exit 1
+fi
+
+VERSION=$1
+
+function check_status () {
+  out=$(mktemp /tmp/osorelease.XXXXX)
+  RES=$(gh api -X GET repos/{owner}/{repo}/actions/runs -f "q='branch:v${VERSION}'" | \
+    jq ".workflow_runs[] | {name, id, conclusion, head_branch} | select((.name | contains(\"Release\")) and (.head_branch | contains(\"${VERSION}\")))" | tee "$out" | jq '.conclusion == "success"')
```

Comment:
also requires folks to have `jq` installed — we have a check in one of our Makefiles to ensure that it's installed

Reply:
yeah